src/components/ga4/EnhancedEcommerce/numeric-input.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react"
+import {MdArrowDropDown, MdArrowDropUp} from "react-icons/md"
+import {decrement, increment, input, wrap} from "./numeric-input.module.css"
+
+export function NumericInput({
+                                 onIncrement,
+                                 onDecrement,
+                                 disabled,
+                                 ...props
+                             }) {
+    return (
+        <div className={wrap}>
+            <input
```

Comment:
@react/mui provides basically all base html elements & the like as pre-made components. I'd recommend using those if possible because it comes with free styling. For this on in particular, I'd recommend using a [button group](https://mui.com/components/button-group/)